src/main/java/RoutineManagerController/WorkoutMenuController.java

```diff
@@ -0,0 +1,35 @@
+package RoutineManagerController;
+
+import RoutineManagerModel.RoutineManager;
+import RoutineManagerView.WorkoutMenu;
+import RoutineManagerView.WorkoutSelection;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class WorkoutMenuController {
+
+    private RoutineManager theModel;
+    private WorkoutMenu theView;
+
+    public WorkoutMenuController(RoutineManager theModel, WorkoutMenu theView){
+
+        this.theModel = theModel;
+
+        this.theView = theView;
+
+        this.theView.addWorkoutSelectionDisplay(new WorkoutSelectionDisplay());
```

Comment:
Since this constructor does have a few parameters, it might be a good idea to add Javadoc indicating what each of parameter is.